frontend/eda/EventDrivenSidebar.tsx

```diff
@@ -5,11 +5,16 @@ import { usePageNavBarClick } from '../../framework/PageNav/PageNavSidebar';
 import { RouteObj } from '../Routes';
 import { CommonSidebar } from '../common/CommonSidebar';
 import { isRouteActive } from '../common/Masthead';
+import { EdaUserInfo } from '../common/Masthead';
 
 export function EventDrivenSidebar() {
   const { t } = useTranslation();
   const location = useLocation();
   const onClick = usePageNavBarClick();
+  const activeUser = EdaUserInfo();
+  const canViewAccess =
```

Comment:
The canViewAccess should also return true is the user is a superuser: activeUser?.is_superuser. The superuser does not have any roles returned.